pkg/cloud/azure/assets/cloud-node-manager-daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: azure-cloud-node-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    component: azure-cloud-node-manager
+    kubernetes.io/cluster-service: "true"
+spec:
+  selector:
+    matchLabels:
+      k8s-app: azure-cloud-node-manager
+  template:
+    metadata:
+      labels:
+        k8s-app: azure-cloud-node-manager
+      annotations:
+        cluster-autoscaler.kubernetes.io/daemonset-pod: "true"
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: cloud-node-manager
+      hostNetwork: true   # required to fetch correct hostname
+      nodeSelector:
+        kubernetes.io/os: linux
+      tolerations:
```

Comment:
same question here, i'm guessing since this uses the host network that it will need to tolerate `not-ready` as well.

Reply:
Host network maybe means we can allow it to tolerate the taint, though IMO, we should only add the toleration if we are certain we need it. I'd suggest being conservative initially